cms/static/js/views/active_video_upload_list.js

```diff
@@ -204,27 +227,77 @@ define([
             },
 
             fileUploadDone: function(event, data) {
-                this.setStatus(data.cid, ActiveVideoUpload.STATUS_COMPLETED);
-                this.setProgress(data.cid, 1);
-                if (this.onFileUploadDone) {
-                    this.onFileUploadDone(this.collection);
-                    this.clearSuccessful();
-                }
+                var model = this.collection.get(data.cid),
+                    self = this;
+
+                this.readMessages([
+                    StringUtils.interpolate(
+                        gettext('Upload completed for video {fileName}'),
+                        {fileName: model.get('fileName')}
+                    )
+                ]);
+
+                this.sendStatusUpdate([
+                    {
+                        edxVideoId: model.get('videoId'),
+                        status: 'upload_completed',
+                        message: 'Uploaded completed'
+                    }
+                ]).done(function() {
+                    self.setStatus(data.cid, ActiveVideoUpload.STATUS_COMPLETED);
```

Comment:
Thanks @muhammad-ammar. I was asking because I don't understand what text change @mushtaqak was proposing in his original comment: "Do we need to change the "In Progress" line from the templates ? and any other changes ?"